src/hostapi/wasapi/pa_win_wasapi.c

```diff
@@ -604,7 +623,7 @@ typedef struct PaWasapiStream
     volatile BOOL isActive;
 
     // Stream state: stopped (triggered by user only via external API)
-    BOOL isStopped;
+    volatile BOOL isStopped;
```

Comment:
Is this change required for AC3 / E-AC3? or is it a fix for a bug that is unrealated to the purpose of this PR? If it is an independent fix please move it to a separate PR so that it is easily seen in the merge history.

Reply:
Although harmless, reverted my change.